langchain/llms/openai.py

```diff
@@ -163,6 +163,9 @@ def _generate(
     def stream(self, prompt: str) -> Generator:
         """Call OpenAI with streaming flag and return the resulting generator.
 
+        BETA: this is a beta feature while we figure out the right abstraction.
+        Once that happens, this interface could change.
```

Comment:
People should also be kind of wary that the interaction with memory isn't ironed out.